api/task_processor/threads.py

```diff
@@ -24,7 +27,10 @@ def __init__(
     def run(self) -> None:
         while not self._stopped:
             self.last_checked_for_tasks = timezone.now()
-            run_tasks(self.queue_pop_size)
+            try:
+                run_tasks(self.queue_pop_size)
+            except Exception as e:
+                logger.exception(e)
             run_recurring_tasks(self.queue_pop_size)
```

Comment:
Do we want to wrap this line too?

Reply:
No, that's fine, this whole try/catch will go away once we figure out the task that is causing the issue